Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,202 @@
+   INFO Consuming Common Manifests from target directory
+   INFO Consuming Openshift Manifests from target directory
+   INFO Credentials loaded from default AWS environment variables
+   INFO Creating infrastructure resources...
+   INFO Waiting up to 20m0s for the Kubernetes API at https://api.cluster-1.openshift-test-1.cilium.rocks:6443...
+   INFO API v1.18.3 up
+   INFO Waiting up to 40m0s for bootstrapping to complete...
+
+Next, firewall configuration update to be updated to allow `Cilium
+ports <https://docs.cilium.io/en/v1.8/install/system_requirements/#firewall-rules>`_.
+Please note that ``openshift-install`` doesn't support custom firewall
+rules, so you will need to use one of the following scripts if you are
+using AWS or GCP, but it's not need on Azure.
+
+.. note:: This has to be done just after ``INFO Waiting up to 40m0s for
+   bootstrapping to complete...`` appears in the logs. It is safe to apply
+   these changes once, OpenShift will not override these.
+
+.. tabs::
+
+   .. group-tab:: AWS: enable Cilium ports 
+
+      .. code:: bash
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         aws_region="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.region')"
+         cluster_tag="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.identifier[0] | to_entries | "Name=tag:\(.[0].key),Values=\(.[0].value)"')"
+
+         worker_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-worker-sg" | jq -r .SecurityGroups[0].GroupId)"
+         master_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-master-sg" | jq -r .SecurityGroups[0].GroupId)"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${worker_sg}"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${master_sg}"
+
+   .. group-tab:: GCP: enable Cilium ports
+
+      .. code:: bash       
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         gcp_projectID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.gcp.projectID')"
+
+         gcloud compute firewall-rules create \
+            --project="${gcp_projectID}" \
+            --network="${infraID}-network" \
+            --allow=tcp:4240,udp:8472,icmp \
```

Comment:
I'm curious, this opens the health TCP port 4240, does ICMP just work by default or is it denied? Wondering how that might impact Cilium's report of cluster connectivity.

Reply:
Personally, I don't know, from [the docs](https://docs.cilium.io/en/v1.8/install/system_requirements/#firewall-rules) it sounds as if all ports must be open.

Reply:
What are you referring to for "all ports must be open"? In the link you provided, there are specific rows to cover ICMP echo packets.

Reply:
Ok, so what I mean is that the way I read that table is that 8472/udp, 4240/tcp and ICMP 8/0 have to be allowed for Cilium to work. In this case ICMP traffic was already allowed, so only 8472/udp and 4240/tcp need to be take care of. Are you saying that I'm misreading the table and ICMP is sufficient for health checks?

Reply:
> Ok, so what I mean is that the way I read that table is that 8472/udp, 4240/tcp and ICMP 8/0 have to be allowed for Cilium to work.

:+1: 

>  In this case ICMP traffic was already allowed, so only 8472/udp and 4240/tcp need to be take care of.

Great, then no change is necessary.